fixed_SCALEConfig.cmake

```diff
@@ -0,0 +1,250 @@
+# Dependencies
+#-----------------------------------------------------------------------------#
+
+include(CMakeFindDependencyMacro)
+
+enable_language(C)
+
+find_dependency(HDF5 COMPONENTS C HL)
+find_dependency(PythonInterp)
+if(SCALE_DATA_DIR)
+  find_dependency(SCALEData)
+endif()
+
+if(SCALE_USE_MPI)
+  find_dependency(MPI COMPONENTS C CXX)
+endif()
+
+if(NOT DEFINED CMAKE_CXX_EXTENSIONS)
+  # Tell Kokkos to STFU
+  set(CMAKE_CXX_EXTENSIONS "")
+endif()
+find_dependency(Trilinos ${SCALE_Trilinos_VERSION})
+
+find_dependency(BLAS)
+find_dependency(LAPACK)
+
+if(SCALE_BUILD_TESTS)
+  if(SCALE_BUILTIN_GTest)
+    # GTest targets should be installed alongside SCALE
+    include("${SCALE_CMAKE_DIR}/../GTest/GTestConfig.cmake")
+  else()
+    if(CMAKE_VERSION VERSION_LESS 3.20)
+      # First look for standard CMake installation
+      # (automatically done for CMake >= 3.20)
+      find_dependency(GTest QUIET NO_MODULE)
+    endif()
+    if(NOT GTest_FOUND)
+      # If not found, try again
+      find_dependency(GTest)
+    endif()
+  endif()
+endif()
+
+if(SCALE_USE_Fortran)
+  find_dependency(Flibcpp)
+endif()
+
+# PugiXML may be installed externally or vendored by SCALE
+if(SCALE_BUILTIN_pugixml)
+  include("${SCALE_CMAKE_DIR}/../pugixml/pugixml-config.cmake")
+else()
+  find_dependency(pugixml)
+endif()
+
+# Celeritas is used by ORANGE
+if(SCALE_BUILTIN_Celeritas)
+  include("${SCALE_CMAKE_DIR}/../Celeritas/CeleritasConfig.cmake")
+else()
+  find_dependency(Celeritas)
+endif()
+
+# GNDS is used by AMPX and may be installed externally or vendored by SCALE
+if(SCALE_BUILTIN_GNDS)
+  include("${SCALE_CMAKE_DIR}/../GNDS/GNDSConfig.cmake")
+else()
+  find_dependency(GNDS)
+endif()
+
+# SuperLU is used by Origen and may be installed externally or vendored by SCALE
+if(SCALE_BUILTIN_superlu)
+  # SuperLU should be installed alongside SCALE
+  include("${SCALE_CMAKE_DIR}/../superlu/superluConfig.cmake")
+else()
+  find_dependency(superlu)
+endif()
+
+# gcem is used by statslib and may be installed externally or vendored by SCALE
+if(NOT SCALE_BUILTIN_gcem)
+  find_dependency(gcem)
+endif()
+
+if(SCALE_USE_MCNP)
+  if(SCALE_BUILTIN_Lava)
+    # Lava will be installed alongside SCALE
+    include("${SCALE_CMAKE_DIR}/../Lava/LavaConfig.cmake")
+  else()
+    find_dependency(Lava)
+  endif()
+endif()
+
+#-----------------------------------------------------------------------------#
+# Targets
+#-----------------------------------------------------------------------------#
+
+if(NOT TARGET SCALE::nemesis)
+  include("${SCALE_CMAKE_DIR}/SCALETargets.cmake")
+endif()
+
+#-----------------------------------------------------------------------------#
+
+check_required_components(SCALE)
+set(SCALE_FOUND TRUE)
+foreach(comp ${SCALE_FIND_COMPONENTS})
+  list(APPEND SCALE_LIBRARIES ${CMAKE_CURRENT_LIST_DIR}/../../lib${comp}.so)
+endforeach()
+set(SCALE_INCLUDE_DIRS ${CMAKE_CURRENT_LIST_DIR}/../../../include)
```

Comment:
Lines 244-248 of this file must be added to the end of `cmake/ScaleInstallSetup/SCALEConfig.modern.cmake.in` in SCALE to get things configuring.